### PR TITLE
less: updated to v650

### PR DESCRIPTION
**Pull request by kenmays**
Bumped to less v650.
- Tested build on Haiku R1B4 x86 (GCC13). Passed unit testing
- Tested build on Haiku R1B4 x64 (GCC13). Passed unit testing

### CODE REVIEW

**Comment by Begasus** [comment]
Would still be nice to have this bumped on 32bit too, for gcc2 there are some parse errors, one I can't seem to tackle.
Maybe switch to secondary architecture and use $prefix/bin to install the binaries for 32bit?

**Comment by kenmays** [comment]
My focus is to update x64 first with this merge..

**Comment by Begasus** [comment]
This builds perfectly for secondary architecture, with the binaries in $prefix/bin this should be OK. A REPLACES will be in order then, any thoughts from the others?

File: sys-apps/less/less-650.recipe [hunk 1]
```diff
@@ -48,7 +49,7 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	make -f Makefile.aut distfiles
-	runConfigure ./configure --with-editor=nano \
+	runConfigure ./configure --prefix=$prefix --with-editor=nano \
```
Comment (Begasus) [inline sys-apps/less/less-650.recipe:52]: prefix not needed here, that's done by runConfigure, please use commandBinDir here.

**Comment by kenmays** [comment]
NOTE: This PR is now archived for historical purposes. Less was recently updated to v643 for Haikuports (i.e. less v643 (12 Aug 2023 release) is the current recommended stable version for general use from the less developers). Less v653 is the current BETA version for testing purposes as of Apr 13, 2024.

See: https://github.com/haikuports/haikuports/pull/10327

**Pull request closed without merging**